config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-periodics.yaml

```diff
@@ -148,64 +148,10 @@ periodics:
             memory: "4Gi"
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-aws
-    testgrid-tab-name: periodic-e2e-conformance-release-0-4
+    testgrid-tab-name: periodic-e2e-conformance-v1alpha2
     testgrid-alert-email: kubernetes-sig-cluster-lifecycle-cluster-api-alerts@googlegroups.com
     testgrid-num-failures-to-alert: "2"
-- name: ci-cluster-api-provider-aws-make-conformance-master
```

Comment:
The ginkgo/sonobuoy-based conformance test seems to be doing good, so switching to that for testing against a known and released version of k8s.